services/src/main/java/org/keycloak/authentication/authenticators/browser/WebAuthnPasswordlessAuthenticatorFactory.java

```diff
@@ -56,7 +56,7 @@ public String getId() {
 
     @Override
     public boolean isUserSetupAllowed() {
-        return false;
+        return true;
```

Comment:
I did not review whole PR in details, but is it possible to remove this change from this PR?

It was recently added on purpose due the security reasons (WebAuthn passwordless is considered "1st factor authenticator" and user is not supposed to be able to setup it during authentication unlike 2nd factor mechanisms).

Let me know if you want more details and we can find the corresponding discussion (I figured the commit message doesn't have JIRA number and hence might be a little bit complicated to find it...)

Reply:
@mposolda Oh, you're right. I remember now something was discussed before. Thanks!